deps/rabbit/src/rabbit_msg_store_gc.erl

```diff
@@ -51,6 +51,9 @@ delete(Server, File) ->
 stop(Server) ->
     gen_server2:call(Server, stop, infinity).
 
+prioritise_cast({delete, _}, _Len, _State) -> 5;
+prioritise_cast(_, _Len, _State) -> 0.
```

Comment:
Not a bad idea. Worth adding a comment that in the future we should switch to using priority messages introduced in OTP-28. Then we can just remove gen_server2 from that process.